agr_api/src/main/java/org/alliancegenome/api/controller/VariantController.java

```diff
@@ -35,7 +35,7 @@
         long startTime = System.currentTimeMillis();
         Pagination pagination = new Pagination(page, limit, sortBy, null);
         pagination.addFieldFilter(FieldFilter.VARIANT_TYPE, transcriptType);
-        pagination.addFieldFilter(FieldFilter.VARIANT_CONSEQUENCE, consequence);
+        pagination.addFieldFilter(FieldFilter.MOLECULAR_CONSEQUENCE, consequence);
```

Comment:
I would guess that `consequence` should be `molecularConsequence`?

Reply:
Yes, let's make it consistent.